packages/squashfs.rb

```diff
@@ -0,0 +1,30 @@
+require 'package'
+
+class Squashfs < Package
+  description 'Squashfs is a compressed read-only filesystem for Linux.'
+  homepage 'http://squashfs.sourceforge.net/'
+  version '4.3'
+  source_url 'https://downloads.sourceforge.net/project/squashfs/squashfs/squashfs4.3/squashfs4.3.tar.gz'
+  source_sha256 '0d605512437b1eb800b4736791559295ee5f60177e102e4d4ccd0ee241a5f3f6'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  depends_on 'help2man' => :build
```

Comment:
May also require:
- lz4
- lzo
- xz
- zlib

Reply:
Those are optional in case you want other compression choices.  The default is gzip.  If you want the others, I can certainly add.  It wouldn't hurt.